doc/v2/documentation_data_format.rst

```diff
@@ -1103,6 +1102,52 @@ Conditions and observables that are not applied to a model do not need to be
 valid for that model.
 
 
+.. _v2_initialization_semantics:
+
+Initialization and parameter application
+----------------------------------------
+
+The following describes how the model is initialized and how other PEtab
+parameter values are applied before the simulation starts.
+This procedure is executed for each model and experiment combination
+defined in the PEtab problem.
+
+1. Pre-initialization
+
+   1. Parameters values for parameters that occur in the parameter table are
+      applied to the uninitialized model.
+      For estimated parameters, the respective externally provided values
+      are applied and for non-estimated parameters, the nominal values
+      from the parameter table are applied.
+
+   2. The time is set to the start time of the first period of the given
+      experiment (or 0 if there is no explicit experiment), and any
+      experiment-specific conditions provided for the first period are applied
+      (see :ref:`v2_reinitialization_semantics` for details).
+      Target value expressions for the first period must not refer to model
+      entities that are not also listed in the parameter table.
```

Comment:
Possible to also allow model entities listed in the first period condition changes? This would mean that the order of application of changes now matters.

Reply:
In principle, yes, as long as there aren't any circular dependencies. However, this would be inconsistent with later periods, where the respective symbols evaluate to their value at the end of the previous condition, not to the newly assigned value.